src/main/java/core/basesyntax/RightTriangle.java

```diff
@@ -0,0 +1,28 @@
+package core.basesyntax;
+
+public class RightTriangle extends Figure {
+    private int firstLeg;
+    private int secondLeg;
+
+    public RightTriangle(Color color, int firstLeg, int secondLeg) {
+        super(color);
+        this.firstLeg = firstLeg;
+        this.secondLeg = secondLeg;
+    }
+
+    @Override
+    public double calculateArea() {
+        return Math.round((firstLeg * secondLeg) * 0.5);
```

Comment:
The `calculateArea` method rounds the area, which might not be necessary. Consider returning the precise value of the area without rounding, as rounding can lead to loss of precision.